Makefile

```diff
@@ -36,11 +36,22 @@ lint_shellcheck:
 
 dev: check-env-vars set_aws_account_dev deploy_pipelines ## Deploy Pipelines to Dev Environment
 
+dev-bootstrap: check-env-vars set_aws_account_dev vagrant-deploy
```

Comment:
This Makefile has auto-documenting feature. `help` target parses target comment and generates help message. You are missing comments on the new targets.